ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/inband-mgmt-parent.yml

```diff
@@ -90,6 +92,8 @@ ethernet_interfaces:
   channel_group:
     id: 22
     mode: active
+  mode: access
```

Comment:
I think you also want spanning-tree port-fast here. (And ensure spanning-tree mode is not 'none' on the parent switch)